### ISSUE TITLE
Interagency Onboarding Training (October 2019)

### ISSUE THREAD

**Issue by jb3436** [ICPI/Engagements#75]
### Onboarding Checklist 
1.	Registration 
     - [x] Finalize Registration
     - [x] ICPI@state.gov to send out registration link
     - [ ] Every 2 weeks starting at 6 weeks to training send follow-up reminders
     - [ ] Share registration information with Leads at 4 weeks, 2 weeks, 1 week 
2.	Agenda creation
     - [x] Send first draft to Leads 6 weeks before training 
     - [ ] Set final draft approved by Leads 4 weeks before training 
     - [ ] Send final draft to presentation POCs 4 weeks before training 
     - [ ] Develop final draft with POCs feedback, POC names, locations, date/times 2 weeks before training 
3.   Communication with participants
     - [ ] Registration link sent via icpi@state.gov 
     - [ ] Registration confirmation email 
     - [ ] Final email with final agenda/all logistics info/pre-work/etc (see Github) – 1 week before training
     - [ ] After training- email all materials, feedback forms, etc. (see Github) 
     - [ ] Day-long calendar invites
4. Communication with POCs
     - [ ] Email with draft agenda asking for feedback– 4 weeks before training 
     - [ ] Email with final agenda, POC names, etc- 2 weeks before training 
     - [ ] Calendar invites for individual POCs – 1-2 weeks before training
5.	Logistics 
     - [ ] Sign in sheet
     - [ ] Location reservations – breakout sessions plus other sessions 
     - [ ] Registration tracking (Google forms) 
     - [ ] Debrief questions
     - [ ] Post-training evaluation (Google forms) 
     - [ ] Sharepoint folder organization 
6.   Day-of Logistics 
     - [ ] Organize tables 
     - [ ] Registration/sign-in sheet
     - [ ] Finding people to come provide presentations 
     - [ ] Breakfast logistics 
     - [ ] Turn on/off projectors

**Comment by jleehurst** [comment]
Draft Agenda for Leads @hessionra

**Comment by jleehurst** [comment]
@hessionra 

Agenda Draft Completed
14 registered attendees, all USAID, except 1

**Comment by jleehurst** [comment]
EMAIL TO POCS: 

Dear Onboarding Training POCs,
You’ve heard on the Inbrief a few times that our next Interagency Onboarding training will be held Tuesday and Wednesday, 10/15-10/16. We already have 16 registered attendees!
Most of you assisted in the training last time. Your Leads have given us the OK to include you in this round. This time, the prep will be a little different as it’ll be more around updating the presentations to reflect changes in Dosseirs/COP/MER/data analytics. 
I know some of you are busy making slides for the PVA courses, and these will be a good starting place for the Onboarding training sessions. However, for the onboarding, we want to allow more time for in-depth analysis using the various platforms. Please update your slides, accordingly, but first and foremost, we want the technical sessions to be highly interactive. So, please plan fun breakouts and interactive activities.
Please see below for deadlines for your action items:

ACTION | DUE    DATE | COMMENTS
-- | -- | --
Confirm general availability   for 10/15-10/16 | COB Wednesday 09/18 | Email Jenny directly
Send specific availability   (immovable meetings only) for your assigned day   *If I don’t receive   any schedule conflicts from you, your session will get scheduled assuming   your day is fully available | COB Friday 9/20 | Email Jenny directly
Update and finalize all   presentation/other materials | COB Tuesday 10/01 | Upload on SP Folder


Session | Time | Presenter(s)
-- | -- | --
Check-In | 9:00-9:30AM | Jenny Hurst
Welcome and Review of Online Content | 9:30-10:30AM | Jasmine Buttolph   Jamie Houston
SIMS 4.0 and Panorama Dossier | 10:30AM-11:30PM | Shaylee Mehta,   Melissa Bochnowicz
MER Overview and Clinical Cascade | 11:30-12:30AM | Jamie Houston,   Kimi Sato
Lunch (on your own) |  
Additional Data Streams | 1:30-2:15PM | Sam Straitz   Ana Scholl
Intro. to DATIM | 2:15-3:15PM | Imran Mujawar
BREAK | 3:15-3:30PM |  
MER Structured Dataset Review | 3:30-4PM | Aliyah Abdul-Wakil
Recap | 4PM-4:30PM | Jasmine Buttolph


Session | Time | Presenter(s)
-- | -- | --
Review of Previous Day | 9:00-9:30AM | Jenny Hurst
HTS Analyses | 9:30AM-10:30AM | Shazad Ahmed
BREAK | 10:30AM-10:45AM |  
Treatment Analyses | 10:45AM-11:45PM | Michelle Williams-Sherlock
Lunch (on your own) | 11:45-1PM
Viral Load Analyses | 1-2PM | Anubhuti Mishra
SELECT ONE SESSION FROM:      TB/HIV       DREAMS | 2-3PM | Catherine, Meaghan   Kimi Sato
BREAK | 3-3:15PM |  
SELECT ONE SESSION FROM:      Key populations      OVC | 3:15-4:15PM | Mala Narasimhan   Jenny Hurst   Jasmine Buttolph/Caroline   Clooney
Recap and Closing | 4:15-4:30PM |

**Comment by jb3436** [comment]
@jleehurst 
Updates:

We need to figure out how to add Module 5:  "data streams and systems" presentation to the in person agenda. Catherine, Jamie, and I looked at it and it would be hard to follow virtually. We still want Module 4 to be virtual only. 

**Dates:** 
We are considering two options since we have space constraints at OGAC due to SAB on day 2 (10/16)

Option 1: Change Dates from Tues/Wed to Thurs/Fri of the same week (if conf a/b) is available @hessionra please check 
Option 2: Hold Day 2 of training at peace corps (if available) @jleehurst could you check with Rita if a room is available?

**Comment by HoustonJamie** [comment]
@jb3436 Jasmine i think you mean Module 3 (virtual) and Module 4 (in person), Module 5 is the MER module.

**Comment by jleehurst** [comment]
UPDATED AGENDA @hessionra @jb3436 @HoustonJamie @cnichols3h 



Session | Time | Time (mins) | Presenter(s) | Contact Info
-- | -- | -- | -- | --
Check-In | 9:00-9:30AM | 30 | Jenny   Hurst | jhurst@peacecorps.gov
Welcome   and Review of Online Content | 9:30-10:30AM | 60 | Jasmine Buttolph   Jamie Houston | jbuttolph@peacecorps.gov   vie3@cdc.gov
SIMS   4.0 and Panorama Dossier | 10:30-11:15AM | 45 | Shaylee   Mehta,   Melissa   Bochnowicz | Shaylee.p.mehta.ctr@mail.mil   bochnowiczMA@state.gov
MER   Overview and Clinical Cascade | 11:15-12:00AM | 45 | Jamie Houston,   Kimi Sato | vie3@cdc.gov   ksato@peacecorps.gov
Lunch   (on your own) | 12:00-1:00PM | 60 |  
Data   Systems Overview | 1:00-1:45PM | 45 | Josh Davis   Shazad Ahmad | lqa9@cdc.gov
Additional   Data Streams | 1:45-2:30PM | 45 | Sam   Straitz   Ana   Scholl |  
BREAK | 2:30-2:45PM | 15 |   |  
MER   Structured Dataset Review | 2:45-4:15PM | 75 | Aliyah   Abdul-Wakil | odc3@cdc.gov
Recap | 4:15-4:30PM | 15 | Jasmine Buttolph | jbuttolph@peacecorps.gov
  |   |   |   |   |  


Session | Time |   | Presenter(s) | Contact   Info
-- | -- | -- | -- | --
Review   of Previous Day | 9:00-9:30AM | 30 | Jenny   Hurst | jhurst@peacecorps.gov
Intro   to DATIM | 9:30AM-10:45AM | 75 | Imran Mujawar | Lrz5@cdc.gov
BREAK | 10:45AM-11:00AM | 15 |   |  
Treatment   Analyses | 11:00AM-12:00PM | 60 | Michelle Williams-Sherlock | myy6@cdc.gov
Lunch   (on your own) | 12:00-1PM | 60
SELECT   ONE SESSION FROM:   Viral   Load Analyses   HTS   Analyses | 1-2PM | 60 | Anubhuti Mishra   Shazad Ahmad | nme1@cdc.gov   lqa9@cdc.gov
SELECT   ONE SESSION FROM:      TB/HIV       DREAMS | 2-3PM | 60 | Catherine,   Meaghan   Kimi   Sato | cnichols@usaid.gov   ksato@peacecorps.gov
BREAK | 3-3:15PM | 15 |   |  
SELECT   ONE SESSION FROM:      Key populations      OVC | 3:15-4:15PM | 60 | Mala   Narasimhan   Jenny   Hurst   Jasmine   Buttolph | ovu4@cdc.gov   jhurst@peacecorps.gov   jbuttolph@peacecorps.gov
Recap   and Closing | 4:15-4:30PM | 15 |   | ICPI Leads
  |   |   |   |   |

**Comment by imujawar** [comment]
@jb3436 , @jleehurst , here's the presentation overview and objectives for the <intro. to DATIM> module (Please let me know if this makes sense, and provide suggestions):
- Familiarity with DATIM environment
- General understanding of data entry (how data goes in!)
- DATIM analytics (how to get data out!):
  - Ability to use Pivot Tables, Visualizer, and maps, and create dashboards.
  - Create and use favorites
- Use Genie
- How to report issues and troubleshoot using DATIM support
![image](https://user-images.githubusercontent.com/18144056/65082093-206d2100-d973-11e9-943d-fb6e6f93aa59.png)

**Comment by jleehurst** [comment]
Hi all, 
Please complete the following prioritized modules as soon as possible for transcribing and uploading onto PVA. 

- [ ] 	Module 1: ICPI Overview @jb3436 
- [ ] Module 2: PEPFAR 101 and COP @HoustonJamie @cnichols3h 
- [ ] Module 3: Intro. to Data Streams and Processes @jaliasd 
- [ ] Module 8: Intro. to Panorama @nmcarloni 
- [ ] Module 11: Intro. to Sharepoint @hessionra

**Comment by jleehurst** [comment]
> @jb3436 , @jleehurst , here's the presentation overview and objectives for the <intro. to DATIM> module (Please let me know if this makes sense, and provide suggestions):
> 
> * Familiarity with DATIM environment
> * General understanding of data entry (how data goes in!)
> * DATIM analytics (how to get data out!):
>   
>   * Ability to use Pivot Tables, Visualizer, and maps, and create dashboards.
>   * Create and use favorites
> * Use Genie
> * How to report issues and troubleshoot using DATIM support
>   ![image](https://user-images.githubusercontent.com/18144056/65082093-206d2100-d973-11e9-943d-fb6e6f93aa59.png)

@imujawar I think this looks really good!

**Comment by jleehurst** [comment]
Onboarding Email

Hello,

Thank you for registering for the ICPI Onboarding Training from October 15-16, 2019. This email confirms your place in the training. We look forward to hosting you! Please see below for important details and preparation requirements:

Logistics:
-	Both days will be held at ICPI/OGAC Office at 1800 G St NW, Washington, DC (10th Floor)
-	Both days will start at 9AM (check-in is from 9-9:30AM) and end around 4:30PM
-	Please bring a government ID to show Security to enter the building. Head to the 10th Floor and you will find the check-in table there!
-	Attached is the ICPI Visitor Welcome Guide for other important reference information (including metro access, etc.)
-	Please bring your laptops and mobile hotspots to the training. There is no wifi in the building and we only have a few hotspots.
-	We will share the agenda for the training as we get closer to the dates.

Training Prep
-	All participants are required to complete the ICPI Pre-work course on the PEPFAR Virtual Academy linked here: XXXXXXXXX . This course should take no more than XXXX minutes to complete. We will be tracking completion on our end as this course is critical for success of the Onboarding Training.
-	Please insure you have access to the following systems before you arrive at ICPI. Some of these systems take a few days for account creation so please do this well in advance of the training!
o	Panorama – you can request a new account on the main login in page: https://pepfar-panorama.org
o	DATIM – you can request an account here: https://register.datim.org/ 
o	PEPFAR Sharepoint – you can request account access through https://www.pepfar.net/

Please let us know if you have any questions!  We look forward to meeting you!

Best,
ICPI

**Comment by hessionra** [comment]
Sent to participants 10/3!

**Comment by jleehurst** [comment]
FINAL AGENDA

Session | Time | Presenter(s) | Contact Info
-- | -- | -- | --
Check-In | 9 – 9:30AM | Jenny Hurst | jhurst@peacecorps.gov
Welcome and Review of Online Content (Part I) | 9:30 –   10:15AM | Jasmine   Buttolph   Kashmira   Kale | jbuttolph@peacecorps.gov   neu6@cdc.gov
SIMS 4.0 and Panorama Dossier | 10:15 – 10:45AM | Shaylee Mehta,   Melissa Bochnowicz | Shaylee.p.mehta.ctr@mail.mil   bochnowiczMA@state.gov
BREAK | 10:45 –   11AM
MER Overview and Clinical Cascade | 11AM – 12PM | Jamie Houston   Kimi Sato | vie3@cdc.gov   ksato@peacecorps.gov
Lunch (on your own) | 12 –   12:50PM
Data Systems Overview | 1 – 2PM | Josh Davis   Shazad Ahmed | jodavis@usaid.gov   lqa9@cdc.gov
Additional Data Streams | 2 – 2:45PM | Sam Straitz   Ana Scholl | sstraitz@usaid.gov   adjapovicscholl@usaid.gov
BREAK | 2:45 – 3PM
MER Structured Dataset Review | 3 – 4:15PM | Aliyah   Abdul-Wakil | odc3@cdc.gov
Recap | 4:15 – 4:30PM | Jasmine Buttolph | jbuttolph@peacecorps.gov

Wednesday, October 16
Morning Location: 3rd Floor Conference Room
Breakout Sessions Location: Asia Conference Room and Nelson Mandela Room (both 10th Floor)

Session | Time | Presenter(s) | Contact   Info
-- | -- | -- | --
Review of Previous Day | 9 – 9:30AM | Jenny Hurst | jhurst@peacecorps.gov
Review of Online Content    (Part II) | 9:30 – 10AM | Jasmine Buttolph   Kashmira Kale | jbuttolph@peacecorps.gov neu6@cdc.gov
Intro to DATIM | 10 – 10:45AM | Imran Mujawar   Kristin Roha | Lrz5@cdc.gov   rohakm@state.gov
BREAK | 10:45 – 11AM
Treatment Analyses | 11AM – 12PM | Katy Battey | iyo@cdc.gov
Lunch (on your own) | 12 – 12:50PM
SELECT ONE SESSION FROM:   ·        Viral   Load Analyses (Mandela)   ·        HTS   Analyses (Asia) | 1 – 1:55PM | Anubhuti Mishra   Shazad Ahmed | nme1@cdc.gov   lqa9@cdc.gov
BREAK | 1:55 – 2PM
SELECT ONE SESSION FROM:   ·        TB/HIV   Analyses (Asia)   ·        DREAMS   Analyses (Mandela) | 2 – 2:55PM | Catherine Nichols   Meaghan Peterson   Kimi Sato | cnichols@usaid.gov   odt4@cdc.gov   ksato@peacecorps.gov
BREAK | 2:55 – 3PM
SELECT ONE SESSION FROM:   ·        Key   Pop Analyses (Asia)   ·        OVC   Analyses (Mandela) | 3 – 3:55PM | Mala Narasimhan   Jenny Hurst   Caroline Cooney   Erin Schelar | ovu4@cdc.gov   jhurst@peacecorps.gov   ccooneyca@state.gov   eschelar@usaid.gov
Recap and Closing | 4 – 4:30PM | ICPI Leads |

**Comment by jleehurst** [comment]
Onboarding Feedback Form

ICPI Interagency Onboarding Training
Feedback Form Day 1


Agency: ________________


Did you complete 75% or more of the pre-work?  Yes            No   


If yes, did you find the pre-work helpful?    Yes            No   


Did you attend 3 or more sessions today?    Yes            No     


Please mark Agree, Disagree, or Neutral for the following:
•	Overall, today’s sessions were helpful in increasing my understanding of the topics
        Agree            Disagree           Neutral

•	The training content met my expectations 
        Agree            Disagree           Neutral

•	The trainers were effective and successful at presenting the content
        Agree            Disagree           Neutral

•	Most of the content shared today was new to me
        Agree            Disagree           Neutral

Of the below sessions, which were most useful and helpful to you (1 min, 2 max):
o	Welcome and Review of Online Content
o	SIMS 4.0 and Panorama Dossier
o	MER Overview and Clinical Cascade
o	Data Systems Overview
o	Additional Data Streams
o	MER Structured Dataset Review

Of the below sessions, which were least useful and helpful to you (1 min, 3 max):
o	Welcome and Review of Online Content
o	SIMS 4.0 and Panorama Dossier
o	MER Overview and Clinical Cascade
o	Data Systems Overview
o	Additional Data Streams
o	MER Structured Dataset Review

How can we improve the sessions from today? Any other comments? 
ICPI Interagency Onboarding Training
Feedback Form Day 2


Agency: ________________


Did you attend Day 1 of the training?   Yes            No     


Did you attend 3 or more sessions today?    Yes            No     


Please mark Agree, Disagree, or Neutral for the following:
•	Overall, today’s sessions were helpful in increasing my understanding of the topics
        Agree            Disagree           Neutral

•	The training content met my expectations 
        Agree            Disagree           Neutral

•	The trainers were effective and successful at presenting the content
        Agree            Disagree           Neutral

•	Most of the content shared today was new to me
        Agree            Disagree           Neutral

Of the below sessions, which were most useful and helpful to you (1 min, 2 max):
o	Review of Previous Day
o	Intro to DATIM
o	Treatment Analyses Overview
o	CIRCLE SESSION YOU ATTENDED: Viral Load Analyses or HTS Analyses
o	CIRCLE SESSION YOU ATTENDED: TB/HIV Analyses or DREAMS Analyses
o	CIRCLE SESSION YOU ATTENDED: Key Pop Analyses or OVC Analyses

Of the below sessions, which were least useful and helpful to you (1 min, 3 max):
o	Review of Previous Day
o	Intro to DATIM
o	Treatment Analyses Overview
o	CIRCLE SESSION YOU ATTENDED: Viral Load Analyses or HTS Analyses
o	CIRCLE SESSION YOU ATTENDED: TB/HIV Analyses or DREAMS Analyses
o	CIRCLE SESSION YOU ATTENDED: Key Pop Analyses or OVC Analyses

How can we improve the sessions from today? Any other comments?

**Comment by jleehurst** [comment]
Final Email to Participants

Hi All,

This is a reminder that you are confirmed to attend the ICPI Onboarding Training on Oct 15-16 at the OGAC Office in Washington DC. Please see attached for the agenda and below for Logistics and Training Prep reminders!

Logistics:
•	Both days will be held at ICPI/OGAC Office at 1800 G St NW, Washington, DC (10th Floor)
•	Both days will start at 9AM (check-in is from 9-9:30AM) and end around 4:30PM
•	Please bring a government ID to show Security to enter the building. Head to the 10th Floor and you will find the check-in table there!
•	Attached is the ICPI Visitor Welcome Guide for other important reference information (including metro access, etc.)
•	Please bring your laptops and mobile hotspots to the training. There is no wifi in the building and we only have a few hotspots.
•	We will share the agenda for the training as we get closer to the dates.

Training Prep:
All participants are required to complete the ICPI Pre-work course on the PEPFAR Virtual Academy linked here. This course should take no more than 45 minutes to complete. You should have received a registration email. If you did not, please let us know. We will be tracking completion on our end as this course is critical for success of the Onboarding Training.

Please ensure you have access to the following systems before you arrive at ICPI. Some of these systems take a few days for account creation so please do this well in advance of the training!
-	Panorama – you can request a new account on the main login in page: https://pepfar-panorama.org
-	DATIM – you can request an account here: https://register.datim.org/ 
-	PEPFAR Sharepoint – you can request account access through https://www.pepfar.net/

We look forward to seeing you soon! 

Thanks,
ICPI

**Comment by jleehurst** [comment]
July Virtual Onboarding Email to Attendees  @shazadahmed10 @gsoybel @HoustonJamie @cnichols3h 

Dear ICPI Onboarding Attendee,
 
This email confirms your place in the ICPI Virtual Interagency Onboarding Training on July 23rd, 24th, 27th, and 28th, 2020. We’re looking forward to virtually hosting you! Please see below for important information and preparation for the training. In addition, please find the agenda attached. Please let us know if you have any questions in the meantime. 
 
•	The training will be held from approximately 9am to 1pm, beginning on Thursday, July 23rd and ending on Tuesday, July 28th. 
•	The training will be held virtually in collaboration with Tech Change. 
•	In order to help trouble-shoot technical issues and answer any questions, prior to the training, a virtual Kick-Off for attendees will be held on [ENTER DATE/TIME]
•	ACTION: Please ensure you have access to the following systems before Thursday, July 23rd
o	Panorama – you can request a new account on the main login page: https://pepfar-panorama.org
o	DATIM – you can request an account here: https://register.datim.org/ (If you haven’t logged into your DATIM account for several months, it may be deactivated so please reactive your account!)
o	PEPFAR Sharepoint – you can request account access directly through https://www.pepfar.net/ 
•	REQUIRED ACTION: Onboarding Pre-work—please complete the following required pre-work modules for the training. In addition, you are welcome to complete any modules related to technical work that you are interested in. The onboarding pre-work will take approximate 80 mins. 
o	Required Modules: 
1.	Introduction
2.	Fundamentals Module 1 
3.	ICPI Overview
4.	SharePoint
5.	Data Streams and Processes
6.	Monitoring, Evaluation, and Reporting (MER)
7.	DATIM
8.	Panorama
9.	SIMS
10.	3rd 90
o	Optional Technical Modules 
1.	HTS Analytics
2.	Key Population Analytics
3.	HIV/TB Analytics
4.	Prevention Analytics
5.	Treatment Analytics
6.	MSD Analytics

Please email ICPI@state.gov if you have any questions!
 
Thanks,
ICPI
Interagency Collaborative for Program Improvement (ICPI)
Office of the Global AIDS Coordinator (OGAC)
President’s Emergency Plan for AIDS Relief (PEPFAR)
1800 G Street NW, Washington DC
ICPI@state.gov
